docs/tutorials/parsec-tutorial.md

```diff
@@ -0,0 +1,557 @@
+---
+Authors:
+  - Mahyar
+---
+
+# Tutorial: Run PARSEC Benchmarks with gem5
+
+## Introduction
+In this tutorial, we will use gem5art to create a disk image for PARSEC benchmarks ([PARSEC](https://parsec.cs.princeton.edu/)) and then run these benchmarks using gem5. PARSEC is mainly designed to represent the applications that require vast amount of shared-memory. 
```

Comment:
Nit: It's nice in markdown, latex, and other markup languages to put one sentence on each line. This makes it easier to look at diffs.